js/packages/berty-i18n/locale/zh/messages.json

```diff
@@ -1,9 +1,9 @@
 {
   "time": {
-    "fuzzy-time": "",
-    "fuzzy-month-day": "",
-    "fuzzy-year-month": "",
-    "datetime": "",
+    "fuzzy-time": "时:分",
+    "fuzzy-month-day": "月/日",
+    "fuzzy-year-month": "年/月",
+    "datetime": "年-月-时:分",
```

Comment:
Doesn't this breaks our time lib ?

Reply:
sorry? time lib?

Reply:
Do you set the display time to 12 hours or 24 hours?   In China, most of them are based on 24 hours。If I understand wrong, please understand。

Reply:
> Do you set the display time to 12 hours or 24 hours? In China, most of them are based on 24 hours。If I understand wrong, please understand。thanks

I mean those patterns are used to setup how time is drawn (you are never gonna see `hh/MM` it's gonna be replaced by the time library).